src/csharp/Grpc.Core/ChannelCredentials.cs

```diff
@@ -169,9 +212,36 @@ internal override bool IsComposable
             get { return true; }
         }
 
+        internal override bool HasGcHandle
```

Comment:
there seems to be too much unnecessary boilerplate around the gcHandle, the two newly added properties don't add any value.

Reply:
All of this is now removed.